### PR TITLE
add curler, fix yaml spec for broker

### PR DESCRIPTION
**Pull request by sshaaf**
**Check list**:

References issue: [issue: Quarkus Funqy Knative Events guide #14454](https://github.com/quarkusio/quarkus/issues/14454)
A subsequent pull request is also in process for the guide update [PR](https://github.com/quarkusio/quarkus/pull/14459)

Your pull request:

- [x] targets the `development` branch
- [x] uses the `999-SNAPSHOT` version of Quarkus
- [x] has tests (`mvn clean test`)
- [x] works in native (`mvn clean package -Pnative`)
- [x] makes sure the associated guide is updated

### CODE REVIEW

**Comment by matejvasek** [comment]
Sorry that missed this.

**Comment by matejvasek** [comment]
Shouldn't we also add default broker yaml?
Something like:
```yaml
apiVersion: eventing.knative.dev/v1                                                    
kind: Broker     
metadata:
  name: default
```

File: funqy-quickstarts/funqy-knative-events-quickstart/src/main/k8s/curler.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+kind: Pod
+metadata:
+  labels:
+    run: curler
+  name: curler
+spec:
+  containers:
+    - name: curler
+      image: fedora:33
+      tty: true
```
Comment (matejvasek) [inline funqy-quickstarts/funqy-knative-events-quickstart/src/main/k8s/curler.yaml:11]: nit: missing eol

**Comment by sshaaf** [comment]
> Shouldn't we also add default broker yaml?
> Something like:
> 
> ```yaml
> apiVersion: eventing.knative.dev/v1                                                    
> kind: Broker     
> metadata:
>   name: default
> ```

I have added this to the guide instead: 
```
kn broker create default \
  --namespace knativetutorial
 ```

File: funqy-quickstarts/funqy-knative-events-quickstart/src/main/k8s/curler.yaml (hunk 1 above)
Comment (sshaaf) [reply funqy-quickstarts/funqy-knative-events-quickstart/src/main/k8s/curler.yaml]: added eol.

**Review by matejvasek — APPROVED** [review]

**Review by gsmet — APPROVED** [review]
I rebased and squashed. Let's wait for CI and merge.

**Comment by gsmet** [comment]
Thanks!

**Pull request merged**